code/solutions/resource-dependencies/resource-dependencies-challenge.yaml

```diff
@@ -0,0 +1,37 @@
+AWSTemplateFormatVersion: "2010-09-09"
+
+Description: AWS CloudFormation workshop - Resource Dependencies Challenge Lab (uksb-1q9p31idr).
+
+Parameters:
+  S3BucketName:
+    Description: Enter a unique name for S3 bucket.
+    Type: String
+
+  LatestAmiId:
+    Type: AWS::SSM::Parameter::Value<AWS::EC2::Image::Id>
+    Default: /aws/service/ami-amazon-linux-latest/amzn2-ami-hvm-x86_64-gp2
+
+Resources:
+  Ec2Instance:
+    Type: AWS::EC2::Instance
+    Properties:
+      ImageId: !Ref LatestAmiId
+      InstanceType: t2.micro
+      SecurityGroups:
+        - !Ref InstanceSecurityGroup
+
+  InstanceSecurityGroup:
+    Type: AWS::EC2::SecurityGroup
+    Properties:
+      GroupDescription: Allow http to client host
+      SecurityGroupIngress:
+        - IpProtocol: tcp
+          FromPort: 80
+          ToPort: 80
+          CidrIp: 0.0.0.0/0
```

Comment:
Missing egress rule

Reply:
For consistency I am reusing this [Logic](https://github.com/aws-samples/cfn101-workshop/blob/58eeab9f401dabba5699c29e213c8bc823d5f440/code/solutions/user-data.yaml#L108)

Reply:
Perfect, yes, If we don't know `VPcId`, there is no way to specify egress rules and has to be excluded from the check. Thanks!